data/json/item_groups.json

```diff
@@ -2158,20 +2158,22 @@
       ["UPS_off", 5],
       ["adv_UPS_off", 3],
       ["string_36", 40],
-      ["longbow", 5],
-      ["compbow", 1],
-      ["bow_sight", 10],
-      ["bow_stabilizer", 10],
+      ["longbow", 1],
+      ["compbow", 3],
+      ["bow_sight", 5],
+      ["bow_stabilizer", 5],
       ["arrowrest", 10],
-      ["compositebow", 3],
-      ["shortbow", 5],
-      ["recurbow", 2],
-      ["reflexbow", 2],
+      ["compositebow", 2],
+      ["shortbow", 1],
+      ["recurbow", 4],
       ["vambrace_larmor", 1],
       ["quiver", 3],
       ["hand_crossbow", 1],
-      ["arrow_wood", 7],
-      ["arrow_cf", 5],
+      ["arrow_aluminum", 3],
+      ["arrow_cf", 3],
+	  ["arrow_wood_field", 5],
+      ["arrow_wood_field_heavy", 5],
+	  ["arrow_wood", 10],
```

Comment:
Replace all tabs in the PR with spaces. If you're using notepad++, it has tabs-to-spaces replacement, or you can use regex to do that.